src/assets/js/main.js

```diff
@@ -1,194 +1,199 @@
-//_____open/close new employee form_________________
-function addNewEmployeeButton(){
-    document.getElementById('employeeForm').classList.toggle('hidden');
-    if(document.getElementById('addTitle').classList.contains('hidden')){
+function addNewEmployeeButtonClick(){
+    $('#employeeForm').toggleClass('hidden');
+    getFocus();
+    if($('#addTitle').hasClass('hidden')){
         closeForm();
-        document.getElementById('employeeForm').classList.toggle('hidden');
+        $('#employeeForm').toggleClass('hidden');
     }
 }
-//_____main checkbox changer_______________
-function changeCheckboxesValue(elem){
-    let checkBoxes = document.getElementsByClassName('checkboxData');
-    if(elem.checked){
-        for(let i = 0; i < checkBoxes.length; i++){
-            checkBoxes[i].checked = true;
-        }
+
+function getFocus() {           
+    $("#nameInput").focus();
+}
+
+function changeCheckboxesValue(){
+    if($('#mainCheckbox').prop("checked") == true){
+        $('.checkboxData').each(function(){
+            $(this).prop('checked', true);
+        })
     }
     else{
-        for(let i = 0; i < checkBoxes.length; i++){
-            checkBoxes[i].checked = false;;
-        }
+        $('.checkboxData').each(function(){
+            $(this).prop('checked', false);
+        })
     }
 }
-//_____________delete checked items____________________
+
 function deleteSelectedItems(){
-    let checkboxes = document.getElementsByClassName('checkboxData');
-    let tr = document.getElementsByTagName('tr');
-    for (let i = checkboxes.length-1; i >= 0; i--) {
-        if (checkboxes[i].checked) {
-            tr[i+1].remove();
+    $('.checkboxData').each(function(){
+        if($(this).prop('checked') == true){
+            $(this).closest('tr').remove();
         }
-    }
-    document.getElementById('mainCheckbox').checked = false;
-    closeForm();
+    })
+    $('#mainCheckbox').prop('checked', false);
 }
-//_________reset form to default values____________
+
 function closeForm(){
-    document.getElementById('addTitle').classList.remove('hidden');
-    document.getElementById('editTitle').classList.add('hidden');
-    document.getElementById('addEmployee').classList.remove('hidden');
-    document.getElementById('hidenEditBtn').classList.add('hidden')
-    document.getElementById('employeeForm').classList.add('hidden');
+    $('#addTitle').removeClass('hidden');
+    $('#editTitle').addClass('hidden');
```

Comment:
instead of adding and removing "hidden" class please use $(element).hide() /.show()

Reply:
resolved